docs/source/gbd2020_models/causes/measles/index.rst

```diff
@@ -0,0 +1,416 @@
+
+The non-fatal measles model "primarily leverages the relationship between direct reports of measles case notifications annually released by the World Health Organization (WHO) in the Joint Reporting Form (JRF), modeled estimates of measles-containing-vaccine (MCV) vaccination coverage proportions for doses 1 and 2 (five year rolling mean coverage), and supplementary immunization campaign (SIA) coverage (lagged by 1, 2, 3, 4, and 5 years) to produce global estimates of measles cases" [GBD-2019-Capstone-Appendix-Measles_2020]_ (p. 721). GBD assumes that 50% of measles cases are moderate (disability weight 0.051, 95% CI: 0.032, 0.074) and that 50% are severe (disability weight 0.133, 95% CI: 0.088, 0.19); symptoms of the measles sequelae in GBD include fever, aches/pain, and weakness. GBD assumes a universal 95% attack rate in the absence of vaccination. Measles prevalence was estimated by multiplying incident case estimations by an average case duration of ten days [GBD-2019-Capstone-Appendix-Measles_2020]_.
+
+The GBD 2019 fatal measles estimates were modeled in one of two ways depending on available data quality for a given country. For data-rich countries, the Cause of Death Ensemble model (CODEm) was used, with measles-containing vaccination dose one, healthcare access and quality (HAQ) index, socio-demographic index (SDI), and mean years of education per capita as covariates (NOTE: the maternal care and immunization covariate was used in place of HAQ and SDI covariates in previous GBD cycles). For data-poor countries, a natural history model was used that built off of the GBD non-fatal estimates. The natural history model estimated a case fatality ratio via a model informed by data from a systematic literature search that was updated in 2019. "With the available measles CFR input data, we make location- and year-specific death estimates using a negative binomial model with Socio-demographic Index (SDI) as a country-level covariate, additionally accounting for three indicators (hospital-based or not; outbreak or not; and rural or urban/mixed) as study-level covariates, with country random effects" [GBD-2019-Capstone-Appendix-Measles_2020]_ (page 177).
+
+.. todo::
+
+   Describe enough of the data sources and modeling process to verify that even
+   though  measles can lead to diarrhea or other causes that we include in our
+   Vivarium models, we won't be double counting mortality and morbidity from
+   these causes. For example, a death caused by diarrheal dehydration due to
+   measles should be counted in the GBD as a death due to measles, not as a
+   death due to diarrheal diseases.
+
+   The relationship with vitamin A deficiency may also be important for our
+   models.
+
+   Make sure to check on measles sequelae as well. Our models so far have not paid much attention to the nonfatal side, but it looks like some of the complications can persist well after someone recovers from measles, so maybe that's important to think about.
```

Comment:
There is some really interesting stuff that goes beyond GBD causal framework about how measles can "reset" the immune system and put people at higher risk of a range of diseases later --- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8590458/

Reply:
Thanks! I'll bookmark this for reference and add it into a later commit